slices/apache2.yaml

```diff
@@ -0,0 +1,319 @@
+package: apache2
+
+essential:
+  - apache2_copyright
+
+slices:
+  bins:
+    essential:
+      - apache2_configs
+      - apache2-bin_bins
+      - apache2-data_data
+      - apache2-utils_bins
+      - base-files_bin
+      - coreutils_bins
+      - media-types_data
+      - perl-base_bins
+      - perl-base_modules
+      - procps_bins
+    contents:
+      /usr/sbin/a2disconf:
+      /usr/sbin/a2dismod:
+      /usr/sbin/a2dissite:
+      /usr/sbin/a2enconf:
+      /usr/sbin/a2enmod:
+      /usr/sbin/a2ensite:
+      /usr/sbin/a2query:
+      /usr/sbin/apache2ctl:
+      /usr/sbin/apachectl:
+
+  configs:
```

Comment:
Also, this slice should be named `config` not `configs` to be consistent with other existing SDFs.